### PR TITLE
Unbreak -DAVIF_BUILD_GDK_PIXBUF=ON build

### PR DESCRIPTION
**Pull request by jbeich**
Fix #1509 regression and apply part of #296 still required on BSDs.

### CODE REVIEW

File: contrib/gdk-pixbuf/CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ if(AVIF_BUILD_GDK_PIXBUF)
             add_library(pixbufloader-avif MODULE ${GDK_PIXBUF_SRCS})
 
             target_link_libraries(pixbufloader-avif PUBLIC ${GDK_PIXBUF_LIBRARIES} avif)
+            target_link_directories(pixbufloader-avif PUBLIC ${GDK_PIXBUF_LIBRARY_DIRS})
```
Comment (wantehchang) [inline contrib/gdk-pixbuf/CMakeLists.txt:14]: @novomesk Daniel, could you take a look at this change?

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks you for the fix. I'm sorry I missed gdk-pixbuf.

**Comment by jbeich** [comment]
CC @novomesk

**Comment by wantehchang** [comment]
We will create a libavif 1.0.1 release to fix the -DAVIF_BUILD_GDK_PIXBUF=ON build.

File: contrib/gdk-pixbuf/CMakeLists.txt (hunk 1 above)
Comment (novomesk) [reply contrib/gdk-pixbuf/CMakeLists.txt:14]: OK for me.

**Pull request merged**